.eslintrc

```diff
@@ -4,9 +4,6 @@
     "jasmine": true,
     "node": true
   },
-  "parserOptions": {
-    "sourceType": "module"
-  },
```

Comment:
This is for ES modules. We're just commonjs modules so we shouldn't have this. This affects whether `'use strict';` is required.